src/ContextMenuTrigger.js

```diff
@@ -50,6 +51,8 @@ export default class ContextMenuTrigger extends Component {
         if (this.props.holdToDisplay >= 0) {
             event.persist();
 
+            this.touchHandled = true;
```

Comment:
This should be inside the lambda function at line 57, otherwise it'll always be true not just when the touch actually triggered the "hold-to-display" functionality, and we're back at the point where preventDefault fires every time.